aszczachor/src/components/navigation/ExpandButton.js

```diff
@@ -0,0 +1,29 @@
+import expandArrow from "assets/mobileMenu/expand-arrow.png";
+import collapseArrow from "assets/mobileMenu/collapse-arrow.png";
+import { Link } from "react-router-dom";
+
+const ExpandButton = ({ label, changeState, isOpen, list, openMenu }) => {
+  return (
+    <>
+      <li key={label} style={{ marginLeft: "17px" }}>
+        <button onClick={changeState}>
+          {label}
+          <img src={isOpen ? collapseArrow : expandArrow} alt="" />
+        </button>
+      </li>
+      {isOpen
+        ? list.map(item => {
```

Comment:
zamiast ternanry możesz użyć logicznego and

Reply:
nie rozumiem do czego tutaj AND użyć bo tu warunek jest?

Reply:
Boolean ? coś : null => Boolean && coś, omijasz tego nulla